lib/cc/engine/analyzers/php/nodes.rb

```diff
@@ -5,7 +5,19 @@ module Engine
     module Analyzers
       module Php
         module Nodes
-          class Node < OpenStruct
+          class Node
+            def initialize(attrs)
+              @attrs = attrs
+            end
+
+            def method_missing(m, *args, &block)
+              if args.length.zero?
```

Comment:
oo missed this comment before pushing up a blunter implementation. Considering it offline now